_includes/component.html

```diff
@@ -13,19 +15,29 @@ <h2 class="usa-alert__heading">{{ page.alert-heading }}</h2>
 </div>
 {% endif %}
 
+<!--Add wcag compliance status tag-->
+{% include accessibility-tests/compliance-tag.html %}
+
 {% if page.parent == null and content.size > 1 %}
 <h2 class="site-component-section-title">About the {{ componentName }} {{ page.type }}</h2>
 <div class="font-lang-md measure-5 margin-top-3 margin-bottom-6">
   {{ content }}
 </div>
 {% endif %}
 
+{% if issues %}
+  {% include known-issues-alert.html %}
+{% endif %}
 {% include component-preview-and-code.html %}
 {% include component-guidance.html %}
-{% if page.component.package %}
-{% include component-package.html %}
+<!--Add accessibility test results-->
```

Comment:
```suggestion
<!-- Add accessibility test results -->
```

Reply:
Fixed in [72386ab](https://github.com/uswds/uswds-site/pull/2138/commits/72386ab6f0ba9f762be1c9f7475a881628d37cb7)